.github/workflows/pipeline-deploy-app-develop.yaml

```diff
@@ -39,7 +39,7 @@ jobs:
           quality_check,
           metadata,
         ]
-      uses: NHSDigital/uec-dos-management/.github/workflows/deploy-infrastructure.yaml@v0.9
+      uses: NHSDigital/uec-dos-management/.github/workflows/deploy-infrastructure.yaml@v0.11
```

Comment:
You may need to rebase from main to pick up change to remove deploy-support-tools-infrastructure job